index.html

```diff
@@ -0,0 +1,317 @@
+              you can see how much you are improving. Built with JavaScript.
+              (The link to the back-end API can be found in the GitHub repo)
+            </p>
+            <ul class="project-stacks card-stacks">
+              <li>HTML</li>
+              <li>CSS</li>
+              <li>JavaScript</li>
+              <li>React</li>
+              <li>redux</li>
+              <li>Jest</li>
+              <li>API</li>
+            </ul>
+            <a
+              href="https://github.com/Spayco/gtime-frontend/"
+              class="card-link link"
+              target="_blank"
+              rel="noreferrer noopener"
+              >See project</a
+            >
+          </div>
+        </div>
+        <div class="card weather">
+          <div class="card-description">
+            <h2>Weather App</h2>
+            <p class="white-description">
+              This is a responsive weather app made with JavaScript that displays the temperature, humidity and timezone for cities around the world.
+            </p>
+            <ul class="project-stacks card-stacks">
+              <li>HTML</li>
+              <li>SASS/CSS</li>
+              <li>JavaScript</li>
+              <li>WebPack</li>
+              <li>Babel</li>
+            </ul>
+            <a
+              href="https://github.com/Spayco/Weather-App/"
+              class="card-link link"
+              target="_blank"
+              rel="noreferrer noopener"
+              >See project</a
+            >
+          </div>
+        </div>
+        <div class="card conference">
+          <div class="card-description">
+            <h2>Conference page</h2>
+            <p class="white-description">
+              This is a sample conference page made using HTML and CSS.
+            </p>
+            <ul class="project-stacks card-stacks">
+              <li>HTML</li>
+              <li>CSS</li>
+              <li>SASS</li>
+            </ul>
+            <a
+              href="https://github.com/Spayco/Conference-page/"
+              class="card-link link"
+              target="_blank"
+              rel="noreferrer noopener"
+              >See project</a
+            >
+          </div>
+        </div>
+        <div class="card katchy">
+          <div class="card-description">
+            <h2>Katchy</h2>
+            <p class="white-description">
+              This is a website for recipes made with Ruby on Rails
+            </p>
+            <ul class="project-stacks card-stacks">
+              <li>Ruby</li>
+              <li>Ruby on Rails</li>
+              <li>HTML</li>
+              <li>CSS</li>
+              <li>ERB</li>
+              <li>RSPec</li>
+            </ul>
+            <a
+              href="https://github.com/Spayco/katchy/"
+              class="card-link link"
+              target="_blank"
+              rel="noreferrer noopener"
+              >See project</a
+            >
+          </div>
+        </div>
+        
```

Comment:
- [ ] It will be great to follow the design provided. Kindly you can use Flexbox to properly align the project and ensure that they look exactly like the provides design. Kindly check below ⬇️ 

**Yours**
![recent work yours](https://user-images.githubusercontent.com/32361963/119725944-371e8780-be79-11eb-911a-66f4ff7ff1bb.PNG)

**Should be**

![recent work original](https://user-images.githubusercontent.com/32361963/119726036-50bfcf00-be79-11eb-9f0d-782a5bab084c.PNG)